src/extension.ts

```diff
@@ -556,22 +572,21 @@ function exposeKubernetes() {
         cmd += ' --port=' + ports[0];
     }
 
-    kubectl.invoke(cmd);
+    runKubectlCommandInTerminal("kubectl-expose", cmd);
```

Comment:
Seem to be using slightly random terminals depending on which command is run and whether it internally goes through the 'active window', 'temp file' or other code path.  All that stuff should be hidden from the user.  If you want commands to share a terminal then maybe have _all_ commands share that terminal; if you want each command to start its own terminal then give them more specific names.  (I think the latter could result in a _lot_ of terminals though.)

Reply:
Nice. I'd like the shared terminal suggestion. 

Besides, i have removed the terminal command for 'temp file', but for normal `apply file`, i'd like to show it in terminal. 

And for other ui operations (get/delete/describe/logs/expose/exec/scale/cluster-info), if it is be done by one single kubectl command, then i'd like to run it in terminal. Otherwise, hide the detail by using the normal `kubectl.invoke`.